shell/platform/windows/flutter_window_unittests.cc

```diff
@@ -377,5 +378,17 @@ TEST(FlutterWindowTest, OnWindowRepaint) {
   win32window.InjectWindowMessage(WM_PAINT, 0, 0);
 }
 
+TEST(FlutterWindowTest, OnThemeChange) {
+  MockFlutterWindow win32window;
+  MockWindowBindingHandlerDelegate delegate;
+  win32window.SetView(&delegate);
+
+  ON_CALL(win32window, GetHighContrastEnabled()).WillByDefault(Return(true));
+  EXPECT_CALL(win32window, GetHighContrastEnabled()).Times(1);
+  EXPECT_CALL(delegate, UpdateHighContrastEnabled(true)).Times(1);
+
+  win32window.InjectWindowMessage(WM_THEMECHANGED, 0, 0);
+}
```

Comment:
This looks good, nice! Could you also add a similar test for the method used at startup?